### PR TITLE
Environment arguments

### LINKED ISSUE #103: Environment variable handling within argparse
Currently we are only using one environment variable to affect the logging level, however we could and should handle it within the LoggerHelper.

My suggestion is that we handle the lookup from the host's environment in the following way:

```
       parser.add_argument(
            "--debug_level",
            action="store",
            default=os.environ.get("WM_DEBUG_LEVEL", None),
            type=str,
            help="Logger debug level",
        )
```

### PR DESCRIPTION
**Pull request by PFigs**
This change allows the arguments to be sourced by the
transport service directly from the environment. 

The environment parameters follow the ones used in 
backend client and wm-config v2.0.0.

Closes #103

To decide/tweak:

- [x] set final env names

### CODE REVIEW

File: python_transport/wirepas_gateway/utils/argument_tools.py [hunk 1]
```diff
@@ -160,126 +161,132 @@ def add_file_settings(self):
...
         )
 
         self.mqtt.add_argument(
             "--mqtt_tls_version",
-            default=ssl.PROTOCOL_TLSv1_2,
+            default=os.environ.get("WM_SERVICES_MQTT_CERT_REQS", ssl.PROTOCOL_TLSv1_2),
             action="store",
             type=str,
-            help=("Specifies the version of the SSL / TLS protocol to be used."),
+            help=("Specifies the version of the  SSL / TLS protocol to be used."),
```
Comment (GwendalRaoul) [inline python_transport/wirepas_gateway/utils/argument_tools.py]: really needed ;-) ?

Comment (PFigs) [reply python_transport/wirepas_gateway/utils/argument_tools.py]: :man_facepalming:

File: python_transport/wirepas_gateway/utils/argument_tools.py [hunk 2]
```diff
@@ -246,15 +246,15 @@ def add_mqtt(self):
 
         self.mqtt.add_argument(
             "--mqtt_tls_version",
-            default=os.environ.get("WM_SERVICES_MQTT_CERT_REQS", ssl.PROTOCOL_TLSv1_2),
+            default=os.environ.get("WM_GW_MQTT_CERT_REQS", ssl.PROTOCOL_TLSv1_2),
```
Comment (GwendalRaoul) [inline python_transport/wirepas_gateway/utils/argument_tools.py]: Should it be WM_GW_TLS_VERSION?

File: python_transport/wirepas_gateway/utils/argument_tools.py [hunk 3]
```diff
@@ -246,15 +246,15 @@ def add_mqtt(self):
...
             action="store",
             type=str,
             help=("Specifies the version of the SSL / TLS protocol to be used."),
         )
 
         self.mqtt.add_argument(
             "--mqtt_ciphers",
-            default=os.environ.get("WM_SERVICES_MQTT_CLIENT_KEY", None),
+            default=os.environ.get("WM_GW_MQTT_CLIENT_KEY", None),
```
Comment (GwendalRaoul) [inline python_transport/wirepas_gateway/utils/argument_tools.py]: Same here

**Pull request merged**